### PR TITLE
CLDC-3481: Add alarm for cloudfront 5xx error rate

### PR DESCRIPTION
**Pull request by RachaelBooth**

### CODE REVIEW

**Comment by RachaelBooth** [comment]
This has less change than it looks!

Annoyingly because of how cloudfront works the new alarm needs to be in us-east-1, and so the topic does too. I've pulled the monitoring module apart a bit because that was what ended up making most sense for sorting out multiple regions of topics.

File: terraform/development/shared/main.tf [hunk 1]
```diff
@@ -187,15 +188,51 @@ module "front_door" {
...
-  create_email_subscription = false
+  prefix = local.prefix
+
+  initial_create = var.initial_create
+}
+
+module "monitoring_topic_main" {
+  source = "../../modules/monitoring_topic"
+
+  create_email_subscription = true
```
Comment (RachaelBooth) [inline terraform/development/shared/main.tf:202]: We actually do have a subscription already, but it's not managed in terraform. I think it should be brought in.

File: terraform/modules/monitoring_secrets/kms.tf [hunk 2]
```diff
@@ -1,27 +1,19 @@
 resource "aws_kms_key" "this" {
-  count = var.create_email_subscription ? 1 : 0
```
Comment (RachaelBooth) [inline terraform/modules/monitoring_secrets/kms.tf:2]: Everything in the module ended up with this condition, so if not wanted the entrypoint should just not call the module.

File: terraform/staging/main.tf [hunk 3]
```diff
@@ -295,15 +296,36 @@ module "networking" {
   vpc_flow_cloudwatch_log_expiration_days = 60
 }
 
-module "monitoring" {
-  source = "../modules/monitoring"
+module "monitoring_secrets" {
+  source = "../modules/monitoring_secrets"
 
-  create_email_subscription = false
```
Comment (RachaelBooth) [inline terraform/staging/main.tf:301]: Again, we did already actually have one just not in terraform.

**Review by kosiakkatrina — APPROVED** [review]

**Pull request merged**